common.js

```diff
@@ -93,6 +93,14 @@ var vcwg = {
       status:   'CG-DRAFT',
       publisher:  'Credentials W3C Community Group'
     },
+    // todo: replace all references to RDF-NORMALIZATION, with this.
+    'RDF-CANON': {
```

Comment:
I don't think there are an RDF-NORMALIZATION references?  Either way, shouldn't need the local RDF-CANON biblio entry since it's in specref: https://www.specref.org/?q=rdf-canon.

Reply:
I removed this.